docs/sftp.rst

```diff
@@ -1,10 +1,51 @@
 SFTP
 ====
 
-The SFTP class allows you to interact with SFTP services.
+The ``SFTP`` class allows you to interact with `SFTP services<https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol>`_,
+using the  `Paramiko SFTP library <http://docs.paramiko.org/en/2.4/api/sftp.html>`_ under the hood.
```

Comment:
It looks like we're actually using version 2.7.1 of Paramiko (http://docs.paramiko.org/en/2.7/api/sftp.html), can you update the link?